toolkit/docs/building/building.md

```diff
@@ -71,48 +76,101 @@ Quick Start
 ## Prerequisites
 Install prerequisites [here](prerequisites.md).
 
-## Build Tools
-By default all build commands are executed from inside the `./toolkit` folder.  A set of GO Based Tools and a Toolchain must be built before building Mariner Packages and Images.
+## Configure the Environment
+By default all build commands are executed from inside the `./toolkit` folder.
 
+A few one-time steps can be run now (they will re-run automatically later if needed)
+
+Remote files are generally only available for release branches.
 ```bash
-cd ~/git/CBL-Mariner/toolkit
-sudo make toolchain REBUILD_TOOLS=y REBUILD_TOOLCHAIN=y DOWNLOAD_SRPMS=y
+# Get the source code
+git clone https://github.com/microsoft/CBL-Mariner.git
+cd CBL-Mariner/toolkit
+
+# Checkout the desired release branch
+git checkout 1.0
+
+# Build the Go tools
+sudo make go-tools REBUILD_TOOLS=y
+
+# Get the package sources
+sudo make input-srpms DOWNLOAD_SRPMS=y
 ```
+**NOTE: All subsequent commands are assumed to be executed from inside the toolkit directory.**
 
-NOTE: A full list of targets and options to sudo make is available [here](#all-build-targets) and [here](#all-build-variables).
+## ISO Quickstart
+```bash
+# Build an ISO version of ./imageconfigs/core-efi.json entirely from downloaded, pre-built packages
+sudo make iso REBUILD_TOOLS=y REBUILD_PACKAGES=n
+```
+
+## VHDX Quickstart
+```bash
+# Build a VHDX of ./imageconfigs/core-efi.json entirely from downloaded, pre-built packages
+sudo make image REBUILD_TOOLS=y REBUILD_PACKAGES=n
+```
+
+## Quick Build from Sources
```

Comment:
I thought I had left a comment here already, but I guess I forgot to complete it.

I like the ISO and VHDX steps above.  Especially the ISO as we discussed offline.
I think that the Quick Build From Source is fine too, but I think it's an "Advanced Topic" and for the average dude, its confusing when there are bootstrapping ideas and full scratch builds presented in the same set of instructions inline.

I want our Slow, Full Build instructions presented as soon as possible after the Quick ISO building instructions because those are the two main scenarios for the community.  They either want to TRY Mariner and they need an ISO, or they want to build their own version of Mariner clean.

Can we restore the "SLOW" end-to-end instructions here.  And then describe "bootstrapping" techniques later.  It is slightly repetitive, but one single approach is presented at a time.

Reply:
Ah, I found the comments. They were on my fork directly and I missed them. I'll shuffle this around.

Reply:
Moved the detailed instructions to another section, just left a one-liner which does a full rebuild.